tests/integration/cairo1_hint_locals/prank/prank_test.cairo

```diff
@@ -0,0 +1,88 @@
+use array::ArrayTrait;
+use result::ResultTrait;
+
+#[test]
+fn test_prank() {
+    let deployed_contract_address = deploy_contract('pranked', ArrayTrait::new()).unwrap();
+
+    let return_data = call(deployed_contract_address, 'get_a', ArrayTrait::new()).unwrap();
+    assert(*return_data.at(0_u32) == 0, 'check call result');
+
+    invoke(deployed_contract_address, 'maybe_set_three', ArrayTrait::new()).unwrap();
+    let return_data = call(deployed_contract_address, 'get_a', ArrayTrait::new()).unwrap();
+    assert(*return_data.at(0_u32) == 2, 'check call result');
+
+    start_prank(123, deployed_contract_address).unwrap();
+
+    invoke(deployed_contract_address, 'maybe_set_three', ArrayTrait::new()).unwrap();
+    let return_data = call(deployed_contract_address, 'get_a', ArrayTrait::new()).unwrap();
+    assert(*return_data.at(0_u32) == 3, 'check call result');
+
+    stop_prank(deployed_contract_address).unwrap();
+
+    invoke(deployed_contract_address, 'maybe_set_three', ArrayTrait::new()).unwrap();
+    let return_data = call(deployed_contract_address, 'get_a', ArrayTrait::new()).unwrap();
+    assert(*return_data.at(0_u32) == 2, 'check call result');
+}
+
+#[test]
+fn test_stop_prank_on_non_existent() {
+    stop_prank(1234).unwrap();
+}
+
+#[test]
+fn test_stop_prank_on_not_pranked() {
```

Comment:
Those two should fail i believe (be broken)

Reply:
We agreed to allow stoping prank on non-pranked/non-existent addreses.